### PR TITLE
Add skeleton of new E2E tests

### PR DESCRIPTION
**Pull request by RothAndrew**

### CODE REVIEW

**Comment by RothAndrew** [comment]
@jeff-mccoy this is the setup (not including the test code, which is just commented out from a different test) that currently exists for adding a new E2E test.

**Comment by RothAndrew** [comment]
99% of it is just copy/pasted from previous tests.

**Comment by RothAndrew** [comment]
Since GitHub Actions only looks at Master this needs to get merged before the actual test gets written since it adds a new GHA Job

File: .github/workflows/test-command.yml [hunk 1]
```diff
@@ -360,6 +360,198 @@ jobs:
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+          GITHUB_STATE: error
+          GITHUB_CONTEXT: "/test e2e - Data Injection"
+          GITHUB_DESCRIPTION: "run cancelled"
+          GITHUB_TARGET_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          GITHUB_REF: ${{ github.event.client_payload.pull_request.head.ref }}
+          GITHUB_OWNER: ${{ github.event.client_payload.github.payload.repository.owner.login }}
+
+  # Run the E2E test of the Data Injection example
```
Comment (YrrepNoj) [inline .github/workflows/test-command.yml:459]: nit: This is a copy of the comment on line 363, it doesn't seem like it's supposed to be.

**Review by YrrepNoj — CHANGES_REQUESTED** [review]
small comment change then it's good to go

**Review by YrrepNoj — DISMISSED** [review]

**Comment by RothAndrew** [comment]
@YrrepNoj one more review please. I had to force push a new commit to fix a GPG signing issue

**Review by YrrepNoj — APPROVED** [review]

**Pull request merged**